daml/Marketplace/Distribution/Auction/Service.daml

```diff
@@ -91,7 +86,7 @@ template Service
           (remainingDepositCidOpt, siCids) <- foldlA generateSettlementInstruction (Some depositCid, []) settleAllocations
 
           -- Transfer remaining deposit back to the Issuer's trading account
-          whenSome remainingDepositCidOpt \depositCid -> (exerciseByKey @AllocationAccountRule allocationAccount.id Allocation.Withdraw with depositCid = depositCid; transferTo = tradingAccount) $> ()
```

Comment:
Improve the comment here: we're not transferring the remaining deposit, but just unlocking it again